src/edit.c

```diff
@@ -4625,7 +4625,7 @@ ins_compl_get_exp(pos_T *ini)
 	     * If 'infercase' is set, don't use 'smartcase' here
 	     */
 	    save_p_scs = p_scs;
-	    if (ins_buf->b_p_inf)
+	    if (ins_buf != NULL && ins_buf->b_p_inf)
```

Comment:
Is this issue found with static analysis, or did you encounter a case where it caused Vim to crash?
If you have a concrete case which causes Vim to crash, it would be good to describe it and or add a test.
An example where Vim crashes will increase severity of the issue, and your chance to get this merged.